.github/workflows/tests.yaml

```diff
@@ -36,8 +36,13 @@ jobs:
       - run: echo "/opt/homebrew/opt/bison/bin:$PATH" >> $GITHUB_PATH
         if: runner.os == 'macOS'
 
-      - run: python -m pip install meson-python ninja pytest
+      - run: python -m pip install meson-python ninja pytest pytest-github-actions-annotate-failures
```

Comment:
I hate `pytest-github-actions-annotate-failures`. Please don't add it

Reply:
any reason you hate it?

Reply:
not try to argue about it, just curious

Reply:
Because it adds noise to the PR that does not help in the slightest to understand why a test failed.